Makefile

```diff
@@ -7,6 +7,12 @@ STABLE_VERSION ?= 1.11
 
 #todo: add BUILD_NUMBER like in anax/Makefile
 
+# If you are building with a container engine different from docker you can set it here (e.g. podman)
```

Comment:
The comments should be clear that these changes are just for convenience to be able to build the container using podman (in case you don't have docker available to you). The issue is not about running it in OCP or CRC, because you can run a docker image in those environments. In fact IBM's product built on open-horizon has been running this SDO container in OCP for a couple years.